### PR TITLE
Added signup Page

### PR DESCRIPTION
**Pull request by rahulcs754**
- added signup page 
- created submit function
- Use API for store user details
 
image

after signup
image

### CODE REVIEW

File: src/Pages/Signup/Signup/SignupForm.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+          </div>
+          <div className="input-box mt-m">
+            <input
+              type="checkbox"
+              className="mr-s"
+              name="accept"
+              onChange={changeHandler}
+              required
+            />
+            I accept all term &amp; condition
```
Comment (vaibhav-mahalle) [inline src/Pages/Signup/Signup/SignupForm.js]: a minor change it should be terms

File: src/index.css [hunk 2]
```diff
@@ -91,6 +91,32 @@
 .pointer {
   cursor: pointer;
 }
+
+.signup-box {
```
Comment (vaibhav-mahalle) [inline src/index.css:95]: you could have used the utility class.

**Review by vaibhav-mahalle — APPROVED** [review]
suggested minor changes, overall noticed good practices followed in the code.

**Review by krishanu2911 — APPROVED** [review]
No Changes Required 
Well written functions and context code loved it
Nice understandable code👍🏻

File: src/index.css (hunk 2 above)
Comment (rahulcs754) [reply src/index.css:95]: sure i will do this after assignment review

**Pull request merged**